explanations/dispatchers.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>I am not sure about the "awkward" terminology. It is just sparse / some combinations of states and choices are invalid. </p><p> </p><p>Isn't it that the spacemap function combines productmap and vmap_1d? Then we should say so without much ado first. Second, a sentence why the need arises (dense / sparse). Third, the example.</p><p> </p><p>Introduce the example with this sentence and remove the last paragraph:</p><p> </p><p>Consider a simplified version of our deterministic test model. Curly brackets {} denote discrete variables; square brackets [] stand for continuous variables.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/lcm/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>